pkgs/development/compilers/symbiflow-yosys/default.nix

```
@@ -0,0 +1,106 @@
+{ stdenv
+, abc-verifier
+, bash
+, bison
+, fetchFromGitHub
+, flex
+, libffi
+, pkgconfig
+, protobuf
+, python3
+, readline
+, tcl
+, verilog
+, zlib
+, plugins ? []
+}:
+
+let
+  localAbc-verifier = abc-verifier.overrideAttrs (_: rec {
+    pname = "abc-verifier";
+    version = "2020.06.22";
+
+    src = fetchFromGitHub {
+      owner  = "YosysHQ";
+      repo   = "abc";
+      rev    = "341db25668f3054c87aa3372c794e180f629af5d";
+      sha256 = "14cgv34vz5ljkcms6nrv19vqws2hs8bgjgffk5q03cbxnm2jxv5s";
+    };
+
+    passthru.rev = src.rev;
+  });
+in
+
+stdenv.mkDerivation rec {
+  pname   = "symbiflow-yosys";
+  version = "0.8-6021-gd8b2d1a2";
+
+  src = fetchFromGitHub {
+    owner  = "SymbiFlow";
+    repo   = "yosys";
+    rev    = "d8b2d1a2b1a93057678cf49bb8f0329f191faba1";
+    sha256 = "1w8jnqzabvzixjllhb6ak2n2gmjvsn6qd996i7z70bsq5rgdkq9g";
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ pkgconfig protobuf flex bison python3 ];
```

```suggestion
  nativeBuildInputs = [ pkg-config protobuf flex bison python3 ];
```
usage of the pkgconfig alias is discouraged for new additions.

---

Makes me wonder why @GrahamcOfBorg evaluation didn't fail for an alias

---

I'm not sure either, it seems to only be in certain cases

---

Well, it's actually not defined in aliases.nix, that's why:
https://github.com/NixOS/nixpkgs/blob/9ecd8455de97b646bdbbd47b269c3660e7022a47/pkgs/top-level/all-packages.nix#L12107

Seems to be from @zimbatm's backlog: abac6afe93a60c6078c7d86b93fbbace4da3b363

---

I'm assuming it would be painful to remove all aliases of it

---

actually, no, it wouldn't....